lib/MumbleConnection.js

```diff
@@ -551,10 +550,10 @@ MumbleConnection.prototype._onUDPTunnel = function( data ) {
     sequence = sequence.value;
     var moreFrames = true;
     while( moreFrames && packet.length > 0 ) {
-
         // Audio frame header.
         var headerLength, frameLength, terminateAudio;
         var header;
+        type = type & 0x7; // Only use the last 3 bits, as all other bits are used for the target which we ignore by now
```

Comment:
Is this really correct or is there a bug somewhere else? The first byte should be:
`( type << 5 | target )` where `target = 0` for normal talk, `1` for direct whisper and `2` for channel whisper.

I know that doesn't make sense because if it was like that then the original implementation wouldn't have worked. :)

So is there a type = type >> 5` somewhere? I'm just curious. I'll take a moment to figure this out but I think the PR itself is fine enough.